autobahn/websocket/protocol.py

```diff
@@ -2146,11 +2146,11 @@ def sendCloseFrame(self, code=None, reasonUtf8=None, isReply=False):
         """
         if self.state == WebSocketProtocol.STATE_CLOSING:
             if self.debugCodePaths:
-                self.factory._log("ignoring sendCloseFrame since connection is closing")
```

Comment:
I  wonder if (hope!) we can just nuke debugCodePaths (and just rely on log.debug)...? @oberstet?